src/java/com/github/jsdossier/CompilerModule.java

```diff
@@ -78,7 +79,7 @@ CompilerOptions provideCompilerOptions(
 
     // IDE mode must be enabled or all of the jsdoc info will be stripped from the AST.
     options.setIdeMode(true);
-    options.setPreserveJsDocWhitespace(true);
+    options.setParseJsDocDocumentation(Config.JsDocParsing.INCLUDE_DESCRIPTIONS_WITH_WHITESPACE);
```

Comment:
You need to update the bazel [WORKSPACE](https://github.com/jleyba/js-dossier/blob/master/WORKSPACE#L31) to use a version of the compiler that includes this API change.